src/mongocxx/exception/error_code.hpp

```diff
@@ -22,6 +22,7 @@ namespace mongocxx {
 MONGOCXX_INLINE_NAMESPACE_BEGIN
 
 enum class error_code : std::int32_t {
+	k_database_uninitialized = 1,
```

Comment:
How about k_invalid_database_object. The error message will clarify the details.